2023-Season/src/main/java/frc/robot/Configuration/Automode.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.Configuration;
+
+public final class Automode {
```

Comment:
This question is for Colin..... Won't this be better as a int base enum. rather than a constant class with constant variables.... just curious

Reply:
I don't believe you can do int based enums in Java. I do agree with changing it to an enum, you can make the update later.